bin/epr

```diff
@@ -161,8 +161,10 @@ def github(user, repo)
       if $body
         body = item.body
         # -3 for "..."
-        body = body.slice(0, DEFAULT_BODY_LENGTH - 3) << "..." if body.size > DEFAULT_BODY_LENGTH
-        rows[-1].insert(4, body)
+        if body
```

Comment:
I think this check should be moved to line 165. Otherwise, if the body is `nil`, then the body column will contain the wrong data.

Reply:
Do you mean line 165 of the original file or the modified one? Because now the whole 
```ruby
body = body.slice(0, DEFAULT_BODY_LENGTH - 3) << "..." if body.size > DEFAULT_BODY_LENGTH
rows[-1].insert(4, body)
```

block gets skipped when body is `nil` and nothing gets written to the body column.

Reply:
> ... block gets skipped when body is nil and nothing gets written to the body column.

Yes, this is a problem. When the body option is given, we [add the column "Body"](https://github.com/sshaw/export-pull-requests/blob/7b8269f6063ed994d82c1130d80afb56ed320f65/bin/epr#L214). If body is `nil` we must still add it otherwise the rows won't match the columns.

Reply:
Ah, true. I have updated the pr. Now the check is done before operating on the body, but if the body is nil, the columns won't get messed up.